docs/pages/guides/webhooks.mdx

```diff
@@ -2,98 +2,296 @@
-to your application database if no users are remaining.
+- [Creating an endpoint](#creating-an-endpoint)
+- [Subscribing to events](#subscribing-to-events)
+- [Security verification](#security-verification)
+- [Replaying events](#replaying-events)
 
-You could also use webhooks to display how many users are currently active in a
-room at any given time. The possibilities are endless, and as time progresses,
-we will include more events for your application to subscribe to.
+### Creating an endpoint [#creating-an-endpoint]
 
-## What is a webhook?
+If you would like to create an endpoint to receive webhook events, you will do
+so from within the webhooks dashboard for your project.
 
-Webhooks pass information from one application to another instantaneously. A
-webhook is simply an HTTP callback triggered by an event. When an application
-subscribes to an event, and that event occurs, the webhook invokes the HTTP
-callback with a URL you specify (called the webhook endpoint). Webhooks
-streamline the collection of data between two connected services in real-time.
+1. From the [dashboard](/dashboard) overview, navigate to the project you would
+   like to add webhooks to
+2. Click on the webhooks tab from the lefthand menu
 
-Like an API, webhooks are a mechanism for a client to receive data. The main
-difference between webhooks and APIs is webhooks function as an inverted API. A
-traditional API receives requests and sends responses, whereas a webhook calls
-an endpoint that you provide where we send events.
+<Figure>
+  <Image
+    layout="responsive"
+    src="/images/docs/webhooks/webhooks-from-dashboard.png"
+    alt="How to find webhooks from the dashboard"
+    width={600}
+    height={335}
+    quality={90}
+  />
+</Figure>
+
+3. Click the “Add Endpoint” button
+
+<Figure>
+  <Image
+    layout="responsive"
+    src="/images/docs/webhooks/add-endpoint-from-dash.png"
+    alt="How to find webhooks from the dashboard"
+    width={600}
+    height={335}
+    quality={90}
+  />
+</Figure>
+
+4. Enter the URL of the endpoint you would like to use and a description of it.
+5. Select the events you would like to subscribe to (if you make no selections,
+   all events will be sent to the endpoint)
+
+<Figure>
+  <video autoPlay loop muted playsInline>
+    <source src="/images/docs/webhooks/create-endpoint.mp4" type="video/mp4" />
+  </video>
+</Figure>
+
+<Banner title="CSRF Protection">
+
+Before sending events, you should disable CSRF protection for the endpoint if it
+is enabled by default. In addition, you should also follow the steps in the
+security verification section before sending events.
+
+</Banner>
+
+### Subscribing to events [#subscribing-to-events]
+
+Each endpoint you provide in the webhooks dashboard listens to all events by
+default but can be easily configured to only listen to a subset by updating the
+Message Filtering section after endpoint creation.
```

Comment:
```suggestion
By default, each endpoint provided in the webhooks dashboard listens to all events. However, you can easily configure it to only listen to a subset of events by updating the "Message Filtering" section after creating the endpoint.
```

The phrase was quite long, could we make it shorter?

Also, not sure if that’s better, but shouldn't we have the section name in quotes? Like we do when referring to buttons?

Reply:
The changes make the phrase even longer, no? And what about making the "Message Filtering" section in quotes to be coherent when we talk about CTAs?